frontends/p4/fromv1.0/converters.cpp

```diff
@@ -1369,8 +1376,10 @@ Converter::Converter() {
     passes.emplace_back(new ComputeTableCallGraph(structure));
     passes.emplace_back(new Rewriter(structure));
     passes.emplace_back(new FixExtracts(structure));
-    passes.emplace_back(new FixMultiEntryPoint(structure));
-    passes.emplace_back(new MoveIntrinsicMetadata(structure));
+    if (CompilerOptions().arch == "v1model") {
```

Comment:
This is a bit strange, to allocate here a new set of Options.

Reply:
Agreed. These two passes assumes v1model, yet are currently implemented in frontend.